src/app/FakeLib/MSBuildHelper.fs

```diff
@@ -446,7 +464,10 @@ let MSBuildWithProjectProperties outputPath (targets : string) (properties : (st
             |> Set.unionMany
 
     let setBuildParam project projectParams =
-        { projectParams with Targets = targets |> split ';' |> List.filter ((<>) ""); Properties = projectParams.Properties @ properties project }
+        { projectParams with 
+            Targets = targets |> split ';' |> List.filter ((<>) "")
+            Properties = projectParams.Properties @ properties project
+            MaxCpuCount = Some ( Some (EnvironmentHelper.getMachineEnvironment().ProcessorCount)) }
```

Comment:
I noticed that the FAKE build itself, which uses `MSBuildWithDefaults` doesn't build with multiple cores because the CPU Count isn't provided on this very flow.